### PR TITLE
Add the blog / website URL attribute

### LINKED ISSUE #188: Consider adding 'blog' pref
Might be nice to have a blog pref and have it assume https:// (since we don't want to use http in this day and age)

### PR DESCRIPTION
**Pull request by ryanlerch**
This adds the new blog / website attribute to the UI and forms.

Requires: fedora-infra/freeipa-fas#106

Resolves: #188

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

### CODE REVIEW

File: noggin/templates/user.html [hunk 1]
```diff
@@ -59,6 +59,12 @@ <h5 class="mb-2" id="user_username">{{ user.username }}</h5>
                   </div>
                 </li>
               {% endif %}
+              {% if user.website_url %}
+                <li class="list-group-item d-flex justify-content-between align-items-center">
+                  <strong title='{{_("RHBZ")}}'><i class="fa fa-fw fa-link" aria-hidden="true"></i> {{_("Website")}}</strong> 
+                  {{ user.website_url }}
```
Comment (abompard) [inline noggin/templates/user.html]: What about making a pop-out link here?

Comment (ryanlerch) [reply noggin/templates/user.html]: done! yeah i forgot to make this a link!

**Review by Conan-Kudo — APPROVED** [review]

**Comment by ryanlerch** [comment]
fedora-infra/freeipa-fas#106 is now merged, so this one is ready to go!

**Comment by ryanlerch** [comment]
Conflict resolved. Just waiting on the CI tests to run again now!

File: noggin/templates/user.html [hunk 2]
```diff
@@ -59,6 +59,12 @@ <h5 class="mb-2" id="user_username">{{ user.username }}</h5>
                   </div>
                 </li>
               {% endif %}
+              {% if user.website_url %}
+                <li class="list-group-item d-flex justify-content-between align-items-center">
+                  <strong title='{{_("RHBZ")}}'><i class="fa fa-fw fa-link" aria-hidden="true"></i> {{_("Website")}}</strong>
```
Comment (abompard) [inline noggin/templates/user.html]: The RHBZ title here seems wrong.

**Review by abompard — CHANGES_REQUESTED** [review]

**Review by abompard — APPROVED** [review]

**Pull request merged**